org.gramps_project.Gramps.yml

```diff
@@ -154,15 +141,15 @@ modules:
         url:  https://gitlab.gnome.org/GNOME/geocode-glib/-/archive/master/geocode-glib-master.tar.gz
         sha256:  9ec6747d31fee70e5b7ede079f63402b67c21c329ade1d8a8ef9c76de5136901
```

Comment:
geocode glib master changed 2 days ago on March 16, which changed the sha256sum. So the sha256sum has to be changed here to compile and push this update for the Gramps flatpak.

Reply:
I changed the URL to a permanent link